src/main/java/org/humancellatlas/ingest/project/Project.java

```diff
@@ -59,6 +59,12 @@
     @Setter
     private String primaryWrangler;
 
+    @Setter
+    private Object publishedToCatalogue;
```

Comment:
Should this be a boolean type? If i remember correctly, this is to store if the user wants the project to be published or not, so I think a better name could be publishToCatalogue ?

Reply:
I think it's to store if the project is published right? It will be unpublished if it is set to false. It's not like an "export" flag since setting it to false will unpublish it

Reply:
But yes it should be a boolean

Reply:
hmm actually publishToCatalogue is not a good idea, it sounds like a verb as well. Ignore that suggestion then.